release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2643,3 +2643,17 @@ link:https://access.redhat.com/solutions/6324771[{product-title} 4.8.11 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+// TODO: Add this to the end of the appropriate 4.8 z-stream RNs that this check is added with.
+// TODO: Do we need to include this notice in all future z-stream RN as well?
+[id="ocp-4-8-preparing-upgrade-next-release"]
+==== Preparing to upgrade to the next {product-title} release
+
+// TODO: replace "TODO" with appropriate z-stream, and is "check" the correct term?
+{product-title} 4.8.TODO introduced a check that will impact upgrading to the next {product-title} release, which is currently planned to be {product-title} 4.9. This is because Kubernetes 1.22, which {product-title} 4.9 is expected to use, removed a significant number of deprecated `v1beta1` APIs.
```

Comment:
"which is currently planned to be" is a pretty solid softening for an explicit 4.9, but possibly worth following whatever pattern comes out of #36262?

Reply:
Yeah, I'll run this all past my doc team before we use this phrasing. I think that this will get wordy/confusing if we don't mention 4.9 at all. I'm hoping the "planned"/"expected" phrasing will be sufficient to allow us to say it. But I'll get confirmation on this before we merge it.

Reply:
Also, `1.22...4.9 is expected to use` should be definitive. We know 4.9 uses kube 1.22.

Reply:
@jottofar the problem is that 4.9 doesn't technically exist yet since it's not GA yet. 

I know this language is a bit offputting, but when we document something in the docs, we're held to it. The upside though is that we can update this language, in the kbase and even here in the z-stream release notes, once 4.9 GAs. So it's only going to ready weirdly from the time when we publish this, until shortly after 4.9 GAs.

Reply:
Changing the 4.9 to {product-version} would be fine in the 4.9 branch. But if this content is being reused across multiple branches, it could result in maintenance challenges.